kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md

```diff
@@ -160,18 +161,28 @@ spec:
   # If the domain home is in the image
   domainHomeInImage: true
   # The Operator currently does not support other images
-  image: "domain-home-in-image:latest"
-  # imagePullPolicy defaults to "Never"
-  imagePullPolicy: "Never"
+  image: "domain-home-in-image"
+  # imagePullPolicy defaults to "Always" if image version is :latest
+  imagePullPolicy: "IfNotPresent"
+  # Identify which Secret contains the credentials for pulling an image
+  #imagePullSecrets:
+  #- name:
   # Identify which Secret contains the WebLogic Admin credentials (note that there is an example of
   # how to create that Secret at the end of this file)
   webLogicCredentialsSecret:
     name: domain1-weblogic-credentials
   # Whether to include the server out file into the pod's stdout, default is true
   includeServerOutInPodLog: true
+  # Whether to enable log home
+  # logHomeEnabled: false
+  # The in-pod name of the directory to store the domain, node manager, server logs, and server .out
```

Comment:
The in-pod name of the directory to store the domain, node manager, server logs, and server .out files in. -> The in-pod name of the directory in which to store the domain, Node Manager, server logs, and server.out files.

Rosemary pointed me to make such change in another review.

Reply:
That's true. For some reason, when I saw it here, I thought it was in code comments, so I tend to be less strict, but what I gave you (Simon) is correct.

Reply:
The space before the .out is significant, and the wording implies were storing the domain home.    How about "Location for log and/or .out files from the WebLogic node manager, domain, and servers."

Reply:
My mistake. I thought you were referring to a server.out file.

Reply:
The whole thing needs to be reworded to reflect the latest default behavior.   
 "Location for domain log, server log, server out, and node manager log files.   'Ignored (logs remain in their original configured location) when logHomeEnabled resolves to false, or if logHome is left unset and domainHomeInImage is true.  Defaults to /shared/logs/DOMAIN_UID when logHomeEnabled is true and domainHomeInImage is false."